maplibre-adapter/src/androidTest/java/com/amplifyframework/geo/maplibre/MapViewTestActivityTest.kt

```diff
@@ -70,6 +70,7 @@ class MapViewTestActivityTest {
                 activity.mapView.setStyle { style ->
                     continuation.resume(style)
                 }
+                signOutFromCognito(activity.auth)
```

Comment:
I think we should investigate the problem with auth SM flow or test case. Signing out after every case is not the best solution since tests were passing earlier.